manage.py

```diff
@@ -47,16 +48,20 @@ def recreate_db():
 
 
 @manager.option('-n',
-                '--number-users',
+                '--fake-count',
                 default=10,
                 type=int,
                 help='Number of each model type to create',
-                dest='number_users')
-def add_fake_data(number_users):
+                dest='count')
+def add_fake_data(count):
     """
     Adds fake data to the database.
     """
-    User.generate_fake(count=number_users)
+    User.generate_fake(count=count)
+    ZIPCode.generate_fake(count=count)
+    # set a random zip for each user without one
```

Comment:
Nit: Comments begin with an uppercase letters and end with a period for consistency.